lib/checks/shared/filename-is-valid-accessible-name.js

```diff
@@ -0,0 +1,29 @@
+const { text } = axe.commons;
+const possibleImageExtensions = [`.jpg`, `.jpeg`, `.png`, `.gif`, `.tif`];
+
+const accText = text.accessibleText(node).toLowerCase();
+
+/**
+ * Check if accessible name, uses filename (along with file extension)
+ * eg: Download image of Tajmahal.jpg
+ */
+const fileExtnInAccText = accText.match(/\.[0-9a-z]+$/i);
```

Comment:
Image file types don't use numbers

Reply:
this is ok - especially if there is an option for configuring these - but why not simply make the regex the last instance of anything searching backwards from the end of the string, looking for the `.`?

`const fileExtnInAccText = accText.match(/\.(?:.(?!\.))+$/i);`